pkg/grpc/vehicle_signal_decoding.proto

```diff
@@ -26,13 +25,13 @@ message PIDRequests {
 
 message DeviceSetting {
   string template_name = 1;
-  string battery_critical_level_voltage = 2;
-  string safety_cut_out_voltage = 3;
-  string sleep_timer_event_driven_interval = 4;
-  string sleep_timer_event_driven_period = 5;
-  string sleep_timer_inactivity_after_sleep_interval = 6;
-  string sleep_timer_inactivity_fallback_interval = 7;
-  string wake_trigger_voltage_level = 8;
+  float battery_critical_level_voltage = 2;
+  float safety_cut_out_voltage = 3;
+  float sleep_timer_event_driven_interval = 4;
```

Comment:
are intervals a fraction of second? What's the unit here and other intervals?

Reply:
went with adding `Secs` to property name